.travis.yml

```diff
@@ -1,27 +1,34 @@
-language: python
-sudo: false
+language: minimal
 
-python:
-- 2.7
-- 3.4
-- 3.5
-- 3.6
+sudo: false
 
-# Enable 3.7 without globally enabling sudo and dist: xenial for other build jobs
-# see this issue for updates https://github.com/travis-ci/travis-ci/issues/9815
 matrix:
+  fast_finish: true
   include:
-    - python: 3.7
-      dist: xenial
-      sudo: true
+    - name: "python-3.7"
+      env: PY=3.7
+    - name: "python-3.6"
+      env: PY=3.6
+    - name: "python-2.7"
+      env: PY=2.7
```

Comment:
@ericdill I changed this to use `conda` instead of `pip` to create the test environment.
The main advantage is to avoid that odd bug where we would get an old `attr` on Python 2.7 and 3.6. The disadvantages are that we have to drop Python 3.5 and 3.4.

Reply:
I'm totally fine dropping py34 and py35